templates/pc/README.md

```diff
@@ -0,0 +1,23 @@
+# Overview
+
+This directory provides a RHEL6 based template, to support legacy guest OS
+which require an older pc-i440fx machine type to boot.
+Clusters that were deployed using the HyperConverged Operator do not include
+support for this machine type. To allow Virtual Machines to request this
+machine type, this option should be manually added.
+
+# Update HCO
+
+It is necessary to add the relevant machine type to a list of permitted types.
+This will allow Virtual Machines to request the pc-i440fx machine type.
+
+$ oc annotate --overwrite -n openshift-cnv hco kubevirt-hyperconverged \
```

Comment:
This renders as 4 lines without backslash at the end: https://github.com/RHsyseng/cnv-supplemental-templates/blob/84ada49937c4c09986411211d900f6acadb9f69d/templates/pc/README.md#update-hco

Could you use code block for clarity?

Reply:
oh, of course